ts/components/screens/OperationResultScreenContent.tsx

```diff
@@ -60,8 +60,8 @@ export const ComposedBodyFromArray = ({
   textAlign = "center"
 }: PropsComposedBody) => (
   <Body style={{ textAlign }}>
-    {subtitle.map(({ text, key, ...props }) => (
-      <Body key={key} {...props}>
+    {subtitle.map(({ text, key, ...props }, i) => (
+      <Body key={key ?? i} {...props}>
```

Comment:
By implicitly using the index, we are not safe against another view in the same hierarchy that is constructed dynamically, using the index as key as well.

I suggest that we we append the index to a string, in order to reduce hitting duplicates (e.g., `` `CBFA_${i}` ``

Reply:
Hi @Vangaorth, thank you for this suggestion! I will integrate it as soon as possible!